### PR TITLE
DOC-303-Security-doc-conversion

### PR DESCRIPTION
**Pull request by Siboney1**
This document was converted from a Word file. The goal is to set a baseline for what will be included in the public-facing document. The consensus is that this document is due for a major review as it is based on former models and data diagrams. Also, standards like FIPS needs to be included.

There are five *md files, one *mdx file and one *js file that sits under src/shared/components/common/SecurityWP.js. The rest are image files given generic names as these images will change.  These all sit under the content/docs/11-security/security-images/ dir.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/910#" title="Author self-approved">Siboney1</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @Siboney1. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/11-security.mdx
Comment (ritawatson) [inline content/docs/11-security.mdx:29]: The Spectro Cloud Platform is composed of a multi-layer deployment stack.

**Review by ritawatson — CHANGES_REQUESTED** [review]
I started reviewing and offered some suggestions. I believe Karl will also review.

File: content/docs/11-security.mdx
Comment (karl-cardenas-coding) [inline content/docs/11-security.mdx:17]: The layout feels awkward and frankly unnecessary.  All we need to do in this page is to provide a high-level overview and provide a link to each topic. The current layout will be challenging to migrate when we switch over to a documentation framework.

File: content/docs/11-security/3-tenant-cluster-security.md
Comment (ritawatson) [inline content/docs/11-security/3-tenant-cluster-security.md:99]:
```suggestion
Spectro Cloud allows you to periodically run Kubernetes conformance and compliance tests using Kubebench, Kubehunter, and Sonobuoy based on a schedule you provide.
```

**Review by ritawatson — CHANGES_REQUESTED** [review]
Finished reviewing and would like to go over comments with Resma. Please allow time for Resma to also have a look.

**Comment by karl-cardenas-coding** [comment]
The content in this PR will not reside in the public-facing documentation site but instead, reside internally and be available to customers upon request.

**Pull request closed without merging**
